### DISCUSSION TITLE
Soroban NFTS

### DISCUSSION THREAD

**Discussion by waizaiffi** [stellar/sorobanathon#158] [General]
Hey, thanks alot for such a given oppurtunity, We really enjoyed making these!

0 - https://litemint.com/items/GCJSWBB32R27QZ7TND5FAFOO4MWGIBN3QS7JN4OH5WW6EAJJEQ7JNU4Q/soroban
1- https://litemint.com/items/GBRFOAVAHWY6NGRAADOMDFOXZWLFPZKO76ESRBUG7YAQOZWCZZER4EYC/Soroban
2- https://litemint.com/items/GATSWNZ7M7BOI2NKTZLSX2B7ZPUFA4FHMFBYU5ADHRUMPDOFWJELPM56/Soroban
3- https://litemint.com/items/GBCOUZ52B434W3CZKUKLNNKQP5JXSM5OKQMEELWDFVEII3OWPCR6KZBU/Soroban
4- https://litemint.com/items/GBXNNQG6LYAMSHHROTEKNJMP7OPRAGCVZCYXN7SPM3OLSSJI5JGVMC6G/Soroban
5- https://litemint.com/items/GBHDT6KWUY45EY2U7DVEDMAGWOE2GWGJ3GSUHBGWGT6YAGXTFUHWP6PA/Soroban
6- https://litemint.com/items/GA46NZJUMTJ36UMIB2LQXDU3WXG6EKMNGT53JNFYVG2AZXTZPUDUNKYG/Soroban

**Comment by Stellarhamza** [comment]
Great concept!

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Reply by waizaiffi** [reply]
Done. submitted the form thanks a lot 
for such a great opportunity
